addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1759,6 +1759,17 @@ const SelectPagerUserValueWidget = SelectUserValueWidget.extend({
         this._updatePage();
     },
 
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    enable() {
+        this.$('we-toggler').click();
+    },
```

Comment:
Is this specific to the `SelectPagerUserValueWidget`? I think it should be in the select which it inherits. Also, the toggler is accessible through `this.menuTogglerEl`

Reply:
Right, also it should be clicked on only if it is not active yet